web/client/reducers/mediaEditor.js

```diff
@@ -100,6 +102,13 @@ export default (state = DEFAULT_STATE, action) => {
         case SELECT_ITEM: {
             return set('selected', action.id, state);
         }
+        case SELECT_MAP: {
```

Comment:
You should use loadMediaSuccess instead, storing params: "textSearch" and "resultData" with the items results. You can store only the map selected or all the maps from items. The second is more compatible. 
You can do it by inserting an ehancer that calls loadMediaSuccess when items change. 

Avoid to add specific type method when working with generic resources, in general.

Reply:
textSearch is not needed imo, there is a withFilere enhancers that manages it.

Reply:
by using the second options, there is something to changes, for example actually the GetData request is performed when selecting an item, not before